### PR TITLE
update: patch the pairs fix from darkreader#2129

### PR DESCRIPTION
**Pull request by ashtonx**
fixes issue
https://github.com/darkreader/darkreader/pull/2129#issuecomment-919761982

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4129,7 +4129,7 @@ INVERT
 ._24NNT
 
 CSS
-._2OVuy{
+.LhRk3{
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4132]: Is it possible to change this selector that is less obscure and don't rely on this random "string"?

Comment (ashtonx) [reply src/config/dynamic-theme-fixes.config:4132]: Will try to look for a way, that said webdev and css is black magic for me so if i fail i won't be able to tell for certain.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4129,8 +4129,8 @@ INVERT
 ._24NNT
 
 CSS
-._2OVuy{
-  opacity: 0.3;
+button[aria-disabled="true"] > span {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4132]: Phew, okay this seems okay.

**Review by Gusted — APPROVED** [review]

**Comment by Gusted** [comment]
Thank you!

**Pull request merged**